pkg/utils/apiutil/serverapi/middleware.go

```diff
@@ -129,7 +129,7 @@ func (h *redirector) matchMicroServiceRedirectRules(r *http.Request) (bool, stri
 	for _, rule := range h.microserviceRedirectRules {
 		// Now we only support checking the scheduling service whether it is independent
 		if rule.targetServiceName == mcsutils.SchedulingServiceName {
-			if !h.s.IsServiceIndependent(mcsutils.SchedulingServiceName) {
+			if !h.s.GetRaftCluster().IsServiceIndependent(mcsutils.SchedulingServiceName) {
```

Comment:
Do we need to unify `mcsutils.SchedulingServiceName` and `utils.SchedulingServiceName` as well?

Reply:
https://github.com/tikv/pd/pull/8476 will change it to constant.xxx